DataFormats/FEDRawData/interface/FEDNumbering.h

```diff
@@ -64,6 +64,8 @@ class FEDNumbering {
      MAXDTFEDID = 779,
      MINDTTFFEDID = 780,
      MAXDTTFFEDID = 780,
+     MINDTUROSFEDID = 1369,
+     MAXDTUROSFEDID = 1371,
```

Comment:
Please preserve the ordering by FED numbers. Could you please also update FEDRawData.doc under DataFormats/FEDRawData/doc, thanks!

Reply:
Then perhaps also DataFormats/FEDRawData/src/FEDNumbering.cc should get updated accordingly...

By the way, @mommsen : do you see any issue if the FEDid ranges for dtmuROS overlap with the ones of TriggerUpgradeFEDID? (If so, I think that the TriggerUpgradeFEDID's should move... but this is not my area of competence, and I just ask)

Reply:
@perrotta, good catch. FEDid ranges should indeed not overlap. I suggest to split the generic trigger upgrade FEDid range into more specific ones:
1350 - 1359 RCT
1360 - 1367 CalTrigUp
1368 - 1371 DTuROS
1372 - 1409 TriggerUpgrade (reserved range)

According to the FED mapping, the first FED id assigned to DTuROS is 1368, not 1369. @jfernan2, do you understand this discrepancy.

Note that in EventFilter/RctRawToDigi/plugins/RctRawToDigi.cc, one would need to change 'MINTriggerUpgradeFEDID' to 'MINRCTFEDID'. @jfernan2, can you take care of this, please?

Reply:
@mommsen : what you suggest is a more general issue, not restricted to the DT muROS packer/unpacker. This PR remained around for quite a long time, and now it is self consistent and it can work standalone. Perhaps you can apply those additional fixes to FEDNumbering and RctRawToDigi yourself as soon as this PR gets merged (or even now, modulo some possible later rebase): what do you think?

Reply:
@mommsen DTuROS has booked 1368-1371, but for the moment only the last 3 will be actually used; 1368 is a backup one.
I have implemented your comments concerning DT in FEDNumbering, I can do the same for the Rct file, but I agree with @perrotta that I would prefer you do it to decouple from this PR

Reply:
@mommsen @perrotta I have modified RctRawToDigi to bring back compatibility.
Yes, 1368 needs a different tratment in the code: it won't be used for the moment, hence data won't be packed/unpacked and not monitored in DQM. If I include 1368 in FEDNumbering, all the other changes requested by Perrota in DQM and packer/unpacker using MINDTUROSFEDIDMIN will fail.
Thanks